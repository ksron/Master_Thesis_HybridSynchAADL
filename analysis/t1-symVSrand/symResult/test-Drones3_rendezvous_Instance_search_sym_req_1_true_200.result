SOLUTION=NONE
SEARCHED_STATES=
EXEC_TIME=3600035ms
