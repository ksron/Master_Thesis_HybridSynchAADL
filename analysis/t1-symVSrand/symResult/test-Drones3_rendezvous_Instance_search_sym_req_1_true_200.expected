==========================================
search [1] in TESTER : {initConst || cinitState,0,200} =>+ {B:BoolExp ||
    O:Object,T:Time,200} such that check-sat(B:BoolExp and eval(
    Drones3rendezvousInstance | (((abs(((drone1 . environment) | c[x]) - (
    drone2 . environment) | c[x]) < [[1.0e-2]]) and (abs(((drone1 .
    environment) | c[y]) - (drone2 . environment) | c[y]) < [[1.0e-2]])) or (((
    abs(((drone1 . environment) | c[x]) - (drone3 . environment) | c[x]) < [[
    1.0e-2]]) and (abs(((drone1 . environment) | c[y]) - (drone3 . environment)
    | c[y]) < [[1.0e-2]])) or ((abs(((drone2 . environment) | c[x]) - (drone3 .
    environment) | c[x]) < [[1.0e-2]]) and (abs(((drone2 . environment) | c[y])
    - (drone3 . environment) | c[y]) < [[1.0e-2]])))), O:Object)) = true .
